sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.aest.controller.ts

```diff
@@ -71,4 +76,36 @@ export class EducationProgramOfferingAESTController extends BaseController {
       payload.offeringStatus,
     );
   }
+  /**
+   * Get all offerings that were requested for change and waiting to be approved/declined.
+   * @returns offerings.
+   */
+  @Get("change-requests")
+  async getOfferingChangeRequests(): Promise<OfferingChangeRequestAPIOutDTO[]> {
+    const offerings =
+      await this.programOfferingService.getOfferingChangeRequests();
+
+    return offerings.map((offering) => ({
+      offeringId: offering.id,
+      programId: offering.educationProgram.id,
+      offeringName: offering.name,
+      submittedDate: offering.submittedDate,
+      locationName: offering.institutionLocation.name,
+      institutionName:
+        offering.institutionLocation.institution.legalOperatingName,
+    }));
+  }
+
+  /**
+   * For a given offering which is requested as change
+   * get the summary of it's actual(preceding) offering.
+   * @param offeringId actual offering id.
+   * @returns preceding offering summary.
+   */
+  @Get(":offeringId/precedingOffering")
```

Comment:
`precedingOffering` gives the idea that the offering will be returned when actually it is the summary. Can we include "summary" in the name, for instance, "change-summary" to be more close to the other also names as "change-requests".